src/components/Combobox.vue

```diff
@@ -111,6 +118,9 @@ export default defineComponent({
 
     const inputRef = ref()
 
+    const comboboxButton = ref()
+    const optionsPanelWidth = computed(() => comboboxButton.value?.el.offsetWidth)
```

Comment:
it seems like `optionsPanelWidth` will always be the initially captured value of `el.offsetWidth`. Could this be an issue when the width changes, eg. resizable sidebars?

Reply:
maybe? How would you improve that?

Reply:
one approach would be a `ResizeObserver`? [vueuse](https://vueuse.org/core/useResizeObserver/) has a composable we could use or adapt. 

... however, I initially misread the PR as affecting the positioning of all combobox's. Given that it's additive with the `escapeOverflow` prop, this isn't a blocking issue at all